packages/dx-react-grid/docs/guides/paging.md

```diff
@@ -1,48 +1,58 @@
-# React Grid Data Paging
+# React Grid - Data Paging
 
-## Overview
+The Grid component supports data paging. You can specify the page size and switch pages programmatically or via the Paging Panel's UI controls. The paging state management, Paging Panel rendering, and local paging logic are implemented in the corresponding plugins. You can also configure the Grid to use server-side paging if your data service supports it.
 
-The Grid's paging features are used to divide data into pages on the client side. However, if your data service supports server-side paging, you can handle the Grid's paging state changes (current page, page size) and pass this data to the server. There are also plugins that display paging controls that enable end-user interaction, for instance, page switching.
+## Related Plugins
 
-## Plugin List
+The following plugins implement filtering features:
 
-Several plugins implement sorting features:
-- [PagingState](../reference/paging-state.md)
-- [LocalPaging](../reference/local-paging.md)
-- [PagingPanel](../reference/paging-panel.md)
+- [PagingState](../reference/paging-state.md) - controls the paging state
+- [LocalPaging](../reference/local-paging.md) - implements the paging logic
```

Comment:
I would provide a more specific description like 'performs local data paging' instead of 'implements the paging logic'. Probably it's also applicable to other guides.